spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -140,8 +136,129 @@ definitions:
+      - c_rc:
+          type: double
+          units: m
+          desc: Amplitude of the cosine harmonic correction term to the orbit radius
+      - c_uc:
+          type: double
+          units: rad
+          desc: Amplitude of the cosine harmonic correction term to the argument of latitude
+      - c_us:
+          type: double
+          units: rad
+          desc: Amplitude of the sine harmonic correction term to the argument of latitude
+      - c_ic:
+          type: double
+          units: rad
+          desc: Amplitude of the cosine harmonic correction term to the angle of inclination
+      - c_is:
+          type: double
+          units: rad
+          desc: Amplitude of the sine harmonic correction term to the angle of inclination
+      - dn:
+          type: double
+          units: rad/s
+          desc: Mean motion difference
+      - m0:
+          type: double
+          units: radians
+          desc: Mean anomaly at reference time
+      - ecc:
+          type: double
+          desc: Eccentricity of satellite orbit
+      - sqrta:
+          type: double
+          units: m^(1/2)
+          desc: Square root of the semi-major axis of orbit
+      - omega0:
+          type: double
+          units: rad
+          desc: Longitude of ascending node of orbit plane at weekly epoch
+      - omegadot:
+          type: double
+          units: rad/s
+          desc: Rate of right ascension
+      - w:
+          type: double
+          units: rad
+          desc: Argument of perigee
+      - inc:
+          type: double
+          units: rad
+          desc: Inclination
+      - inc_dot:
+          type: double
+          units: rad/s
+          desc: Inclination first derivative
+      - af0:
+          type: double
+          units: s
+          desc: Polynomial clock correction coefficient (clock bias)
+      - af1:
+          type: double
+          units: s/s
+          desc: Polynomial clock correction coefficient (clock drift)
+      - af2:
+          type: double
+          units: s/s^2
+          desc: Polynomial clock correction coefficient (rate of clock drift)
+      - toe_tow:
+          type: double
+          units: s
+          desc: Time of week
+      - toe_wn:
+          type: u16
+          units: week
+          desc: Week number
+      - toc_tow:
+          type: double
+          units: s
+          desc: Clock reference time of week
+      - toc_wn:
+          type: u16
+          units: week
+          desc: Clock reference week number
+      - valid:
+          type: u8
+          desc: Is valid?
+      - healthy:
+          type: u8
+          desc: Satellite is healthy?
+      - sid:
+          type: u32
+          desc: |
+            Signal identifier being tracked - if the 24 MSB are not set
+            itrepresents a PRN-1 identifier
+      - iode:
+          type: u8
+          desc: Issue of ephemeris data
+      - iodc:
```

Comment:
@henryhallam Do you think the rest of that could be fit into a u32. My thought is "yes".